### PR TITLE
Don't apt-get update, just install

### PR DESCRIPTION
**Pull request by wadevries**
Saves ~10s on setup time, and also installs openvpn correctly.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -23,7 +23,6 @@ runs:
     - name: Install OpenVPN
       shell: bash
       run: |
-          sudo apt-get update
           sudo apt-get --no-install-recommends -qq install openvpn
```
Comment (wadevries) [inline action.yml:26]:
```suggestion
      run: sudo apt-get --no-install-recommends -qq install openvpn
```

**Review by 153957 — APPROVED** [review]
Ok, seems fine, does it now get a different version of openvpn?

**Comment by jaap3** [comment]
Doesn't this mean this action now depends on a (possibly stale) apt cache in the container it happens to run in?

**Comment by wadevries** [comment]
> Doesn't this mean this action now depends on a (possibly stale) apt cache in the container it happens to run in?

I guess... This PR contradicts GitHub's own recommendation: https://docs.github.com/en/actions/using-github-hosted-runners/customizing-github-hosted-runners#installing-software-on-ubuntu-runners. It might prevent us from getting security releases of openvpn.

I'll investigate other approaches to reduce setup time.

**Pull request closed without merging**